lib/components/viewers/pattern-row.js

```diff
@@ -99,7 +101,12 @@ export default class PatternRow extends Component {
                           To {stopTime.headsign}
                         </div>
                         <div className='cell time-column'>
-                          {getFormattedStopTime(stopTime, homeTimezone, stopViewerArriving, timeFormat)}
+                          <StopTimeCell
+                            homeTimezone={homeTimezone}
+                            soonText={stopViewerArriving}
+                            stopTime={stopTime}
+                            timeFormat={timeFormat}
+                          />
                         </div>
                         <div className='cell status-column'>
                           {stopTime.realtimeState === 'UPDATED'
```

Comment:
Since you've created StopTimeCell, I think it might be useful to create a similar component for `getStatusLabel` below.  Also, it would probably be good to combine the `Scheduled` div as part of the new `StatusLabel` component.

Reply:
Let's do that in another PR.

Reply:
Part of my rationale for doing it in this PR is that by moving the other JSX code out of the file, we're making `util/viewer.js` more in line with other util files that don't have any React code. If we just go ahead with moving `getStatusLabel`, we'll complete that effort and make the purpose of the util file a little more clear and it will allow us to remove the `React` import.

Reply:
Two arguments against it:
- Unlike `StopTimeCell`, those changes are about code that was not touched in the course of this PR.
- Those changes will expose DRY opportunities, especially in the `RealTimeTimeColumn` of the `ItineraryBody` component.

What I can do is set up another PR that addresses that, and if we feel the changes are small enough, we can merge that into this PR.

Reply:
The PR for the status label is at https://github.com/opentripplanner/otp-react-redux/pull/294.